### PR TITLE
Add docs for Multicollins.cacheEnabled

### PR DESCRIPTION
**Pull request by Primer42**
Docs for https://github.com/tumblr/collins/pull/437
@tumblr/collins

### CODE REVIEW

**Comment by roymarantz** [comment]
:+1: if it is true

File: _includes/configuration/multicollins.html [hunk 1]
```diff
@@ -50,7 +50,12 @@
     <tr>
       <td class="inlinecode">queryCacheTimeout</td>
       <td>Integer</td>
-      <td>The amount of time (in seconds) to cache the query responses from remote collins instances. Defaults to 30 seconds.</td>
+      <td>The amount of time (in seconds) to cache the query responses from remote collins instances. A value of 0 will result in values never expiring from the cache, which you probably do not want. Defaults to 30 seconds.</td>
+    </tr>
+    <tr>
+      <td class="inlinecode">cacheEnabled</td>
+      <td>Boolean</td>
+      <td>Optionally disable the multicollins cache.  True by default.</td>
```
Comment (defect) [inline _includes/configuration/multicollins.html]: This sentence is a bit confusing :) Is it disabled if true?

Comment (Primer42) [inline _includes/configuration/multicollins.html]: @defect is in better now?

**Comment by defect** [comment]
👍

**Pull request merged**